### PR TITLE
[Minor] Fix typo in `importing_images.rst` ("Rougness" should be "Roughness")

### PR DESCRIPTION
**Pull request by GreentheNinja**

### CODE REVIEW

File: tutorials/assets_pipeline/importing_images.rst [hunk 1]
```diff
@@ -381,7 +381,7 @@ Roughness > Mode
 The color channel to consider as a roughness map in this texture. Only effective if
 **Roughness > Src Normal** is not empty.
 
-Rougness > Src Normal
+Roughness > Src Normal
 ^^^^^^^^^^^^^^^^^^^^^
```
Comment (AThousandShips) [inline tutorials/assets_pipeline/importing_images.rst:385]:
```suggestion
^^^^^^^^^^^^^^^^^^^^^^
```

**Review by AThousandShips — COMMENTED** [review]
The underline must match the heading

When fixing this please do so with `git commit --amend` if you are using the command prompt, or squash your commits into one

File: tutorials/assets_pipeline/importing_images.rst (hunk 1 above)
Comment (GreentheNinja) [reply tutorials/assets_pipeline/importing_images.rst]: Should be fixed, I hope I did that right?

**Review by skyace65 — APPROVED** [review]

**Comment by skyace65** [comment]
Thanks! And congrats on your first merged PR!

**Comment by mhilbrunner** [comment]
Cherry-picked to 4.2.

**Pull request merged**